libs/community/langchain_community/vectorstores/pgvector.py

```diff
@@ -638,6 +638,7 @@ def _results_to_docs_and_scores(self, results: Any) -> List[Tuple[Document, floa
         docs = [
             (
                 Document(
+                    id=result.EmbeddingStore.id if result.EmbeddingStore.id else None,
```

Comment:
How do we know `result.EmbeddingStore` always has an `id` attribute?

For what it's worth I don't see that attribute here:
https://github.com/langchain-ai/langchain/blob/09c2d8faca42f09a924599c9d75cc38e05ba1ad4/libs/community/langchain_community/vectorstores/pgvector.py#L187